### DISCUSSION TITLE
[FEATURE] 登录跳转优化

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [1Panel-dev/1Panel#5892] [Q&A]
### 1Panel 版本

社区版1.10.12-LTS

### 请描述您的需求或者改进建议

目前1Panel无论访问哪个页面,只要检测未登录就会跳转至登录入口。
但与飞致云另外一个产品Halo不同,1Panel在登录后都会返回概览页。
我希望将登录跳转操作改成与Halo一致的,这样使用起来更舒服点。

**Comment by wanghe-fit2cloud** [comment]
麻烦再详细描述一下具体需求（比如：Halo 产品的跳转逻辑）。

**Comment by QingFeng-awa** [comment]
访问https://demo.halo.run/uc/profile
https://demo.halo.run/console/login?redirect_uri={https://demo.halo.run/uc/profile}
